lib/src/eval/compiler/expression/prefix.dart

```diff
@@ -55,26 +59,30 @@ BuiltinValue _oneForType(TypeRef type, CompilerContext ctx) =>
         ? BuiltinValue(intval: 1)
         : BuiltinValue(doubleval: 1.0);
 
-Variable _handleDoubleOperands(PrefixExpression e, CompilerContext ctx) {
-  var V = compileExpression(e.operand, ctx);
-  final ref = compileExpressionAsReference(e.operand, ctx);
-  final L = ref.getValue(ctx);
+Variable _handleDoubleOperands(
+  PrefixExpression e,
+  CompilerContext ctx,
+  Reference V,
+  TypeRef type,
+  Variable L,
+) {
   var out = L;
```

Comment:
This `out` variable is misnamed as you're not returning it. I think you can get rid of it and just use L

Reply:
Simply using `L` is causing:

```plain
dart_eval runtime exception: type 'Null' is not a subtype of type 'num' in type cast
#0      NumAdd.run (package:dart_eval/src/eval/runtime/ops/primitives.dart:93:36)
#1      Runtime.execute (package:dart_eval/src/eval/runtime/runtime.dart:787:12)
#2      Runtime.executeLib (package:dart_eval/src/eval/runtime/runtime.dart:775:12)
```

But anyway, it's misnamed, correct.